app/views/directives/traffic-table.html

```diff
@@ -0,0 +1,84 @@
+<!-- Table combining routes and ports for a service -->
+<table class="table table-bordered table-hover table-mobile" style="{{ hasFooter ? 'margin-bottom:5px;' : ''}}">
+<thead>
+  <tr>
+    <th ng-if="!showNodePorts">{{customNameHeader || 'Route'}}</th>
+    <th ng-if="showNodePorts">{{customNameHeader || 'Route'}} / Node Port</th>
```

Comment:
I'm not sure if this will screw up a screen reader making these different `<th>` elements, when the content can change dynamically, instead I would probably do

```
<th>{{customNameHeader || 'Route'}}<span ng-if="showNodePorts"> / Node Port</span></th>
```